### PR TITLE
Add another virtualbox test; make virt strings translatable

### PR DESCRIPTION
**Pull request by bp0**
Perhaps it will help with #154.

### CODE REVIEW

File: modules/computer.c [hunk 1]
```diff
@@ -403,35 +403,43 @@ gchar *computer_get_virtualization(void)
...
     DEBUG("Detecting virtual machine");
 
     if (g_file_test("/proc/xen", G_FILE_TEST_EXISTS)) {
          DEBUG("/proc/xen found; assuming Xen");
-         return g_strdup("Xen");
+         return g_strdup(_("Virtual (Xen)"));
+    }
+
+    tmp = module_call_method("devices::getMotherboard");
+    if (strstr(tmp, "VirtualBox") != NULL) {
```
Comment (mckaygerhard) [inline modules/computer.c]: umm i cannot remenber if this function are case-sensitive! but this will work!?

**Comment by bp0** [comment]
Yes.

**Comment by mckaygerhard** [comment]
i think this pull are not necesary, and sorry men, errr that pull only makes translatable the strings (more to translate) and added extra checks to model name.., the :
```
    tmp = module_call_method("devices::getMotherboard");
    if (strstr(tmp, "VirtualBox") != NULL) {
```
its the only related change and does not make any sense.. in fact i tested then and still does not work..

**Comment by bp0** [comment]
It works because the DMI board name contains the string "VirtualBox" in VirtualBox.

**Comment by mckaygerhard** [comment]
umm in fact yes! that's right as you can see in the pictures below, but still got unkwon phisical machine.. umm maybe i patch worng,, i'll download and compiles directly from your branch either got the patch

hardinfo-virtualbox

hardinfo-virtualbox-1

**Comment by bp0** [comment]
Board/Name is actually missing in your screencap.

What does it show in English? `LANGUAGE=en hardinfo`
What is the actual board name value from DMI? `cat /sys/devices/virtual/dmi/id/board_name`
What version of VirtualBox is it?

**Comment by mckaygerhard** [comment]
umm right!  as i said board_name its missing in this debian, or i mean, in the virtual machie interaction!?

but as i said are always present in the BIOS report name, so bios_vendor and produc_name are more viable way for sure .. only board name are not enought due in the Blade Vmware does not report enought info..  (i not have access right now but in one week i could test that):
```
root@sysinternet:/home/systemas/bin# ls /sys/devices/virtual/dmi/id/
bios_date     modalias	    product_serial   subsystem
bios_vendor   power	    product_uuid     sys_vendor
bios_version  product_name  product_version  uevent
root@sysinternet:/home/systemas/bin# 
systemas@sysinternet:~/bin$ cat /sys/devices/virtual/dmi/id/bios_vendor 
innotek GmbH
systemas@sysinternet:~/bin$ cat /sys/devices/virtual/dmi/id/product_name 
VirtualBox
systemas@sysinternet:~/bin$ 
```

**Comment by bp0** [comment]
Can you try with combined #157 and #160 ?

**Comment by mckaygerhard** [comment]
umm, its better to test with calm the #160 in few hours and i'll report feedback to you ... here i only have a russian machine with VB 3.2.4, inthe other section of the building have more to test.. gimme some time

**Comment by mckaygerhard** [comment]
ok combined its working so relies on another function in another piece of code, the motherboard info as i suggested.. i true that duplication of code are not good but maybe if the code for DMI board manufacturer can be in same file?

.. also if the groups info already will provide headers so the headers section will have a unconsistent code.. to uinderstand

**Comment by bp0** [comment]
The fix is actually that get_motherboard() will now fallback to dmi/product_name if dmi/board_name is empty. Your very old version of VirtualBox that only provided product_name is now detected.

**Comment by bp0** [comment]
Rebased.

**Comment by lpereira** [comment]
Merged, thanks!

**Pull request merged**